src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -106,8 +107,10 @@ export class ExposureNotificationService {
     this.backendInterface = backendInterface;
     this.storage = storage;
     this.secureStorage = secureStorage;
-    this.exposureStatus.observe(status => {
-      this.storage.setItem(EXPOSURE_STATUS, JSON.stringify(status));
+    this.exposureStatus.observe(async status => {
+      await this.secureStorage.set(EXPOSURE_STATUS, JSON.stringify(status), {
+        accessible: ACCESSIBLE.ALWAYS_THIS_DEVICE_ONLY,
```

Comment:
Just dropping a few links for context and others ability to follow along more easily.

https://developer.apple.com/documentation/security/keychain_services/keychain_items/restricting_keychain_item_accessibility?language=objc

and

https://developer.apple.com/documentation/security/keychain_services/keychain_items/item_attribute_keys_and_values?language=objc#1679100

search "Accessibility Values" as there is no page anchors.

It appears that "kSecAttrAccessibleAlwaysThisDeviceOnly" has been deprecated per -> https://developer.apple.com/documentation/security/ksecattraccessiblealwaysthisdeviceonly?language=objc

I would think we'd want to use kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly as the next option up the list, but appreciate that may introduce a potential problem when trying to get the element on a restart if our background task fires before a user unlocks.

Reply:
That's precisely why I choose `ALWAYS_THIS_DEVICE_ONLY`, to handle unknown behaviour when not unlocking after a reboot. Not sure what the best fix is in this case.